src/utils/calldata/tuple.ts

```diff
@@ -69,6 +91,11 @@ function getClosureOffset(input: string, open: string, close: string): number {
   return Number.POSITIVE_INFINITY;
 }
 
+/**
+ * Extract and parse a Cairo 1 tuple into its components.
```

Comment:
jsdocs are unnecessary on all functions that are not exported! so you can remove here and on other functions above